testing/blip/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Nicola Worthington <nicolaw@tfb.net>
+# Maintainer: Nicola Worthington <nicolaw@tfb.net>
+pkgname=blip
+pkgver=0.10
+pkgrel=0
+pkgdesc="Bash Library for Indolent Programmers"
+url="https://github.com/neechbear/blip"
+arch="noarch"
+license="MIT"
+depends="bash bc grep"
+makedepends="bash"
+install=""
+subpackages="$pkgname-doc"
+source="https://github.com/neechbear/blip/releases/download/v0.10-1/blip-0.10.tar.gz"
```

Comment:
Source should use kind of `https://github.com/neechbear/blip/archive/v${pkgver}.tar.gz`

Reply:
Release and archive downloads for this project aren't identical. I have corrected to include pkgver variables though.